docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -82,11 +100,17 @@
 * For radios with an internal module, lines to put the module into flashing mode to perform pass-through flashing
 * Unused GPIO pins shall be connected with a 10k resistor either to GND or VCC. The initial hw revision of a radio shall have them tied to GND, further revision shall use one or multiple of those pins to mark different hw revision of the radio to allow auto detection of the radio version or variant.
 
+### hardware design guidlines
+* all high speed interfaces like Quad/Octo/Hexa SPI, SDRAM, USB and SDIO shall be length / time of flight equalised
+* possible pin assignments can be found in the documentation of the evaluation board: NUCLEO-H563ZI
+ 
 ### Optional (B/W)
 * Display backlight control via PWM - connection to hw-PWM capable pin on the MCU
 * Gimbals with either analog, PWM (e.g. TLE4998P3), SPI ADC (e.g. ADS7952) or UART communication
 * Switches as digital or analog inputs
 * haptic vibrator, via binary or PWM output
+* support for additional key navigation input. Supported keys: Enter, Return, Page>, Page<, Up/Down, System, Model, Tele.
```

Comment:
* support for additional key navigation input, via keys: Enter, Return, Page>, Page<, Up/Down, System, Model, Tele.